pep8ext_naming.py

```diff
@@ -61,12 +61,23 @@ class NamingChecker(object):
     """Checker of PEP-8 Naming Conventions."""
     name = 'naming'
     version = __version__
+    ignore_names = ['setUp', 'tearDown', ]
 
     def __init__(self, tree, filename):
         self.visitors = BaseASTCheck._checks
         self.parents = deque()
         self._node = tree
 
+    @classmethod
+    def add_options(cls, parser):
+        parser.add_option('--ignore-names', default=[], action='store',
+                          type='string', help="Names that should be ignored.")
+        parser.config_options.append('ignore-names')
+
+    @classmethod
+    def parse_options(cls, options):
+        cls.ignore_names = options.ignore_names.split(' ')
```

Comment:
Shouldn't they be comma-separated? E.g.,

```py
flake8 --ignore-names=setUp,tearDown,fooBar,inlineDeferreds
```

That said, we *could* split on both. Thoughts?